routes/calories.js

```diff
@@ -30,11 +38,11 @@ module.exports = (function() {
     return stuffToDisplay;
   }
 
-  router.get('/time',function(req,res){
+  router.get('/time', require_signin, function(req,res){
     res.render('caloriepage', genContext());
   });
 
-  router.get("/:user_id", (req, res, next) => {
+  router.get("/:user_id", require_signin, (req, res, next) => {
```

Comment:
Note: this is nothing wrong with your code, but /:user_id should not exist, since we should not allow just anyone to view any other person's calorie tracker

Reply:
Likely needs to be renamed

Reply:
Kevin's area of the project. Not touching!